core/src/test/java/com/sap/ai/sdk/core/client/DeploymentUnitTest.java

```diff
@@ -14,23 +14,17 @@
 
 import com.sap.ai.sdk.core.client.model.AiDeployment;
 import com.sap.ai.sdk.core.client.model.AiDeploymentBulkModificationRequest;
-import com.sap.ai.sdk.core.client.model.AiDeploymentBulkModificationResponse;
 import com.sap.ai.sdk.core.client.model.AiDeploymentCreationRequest;
-import com.sap.ai.sdk.core.client.model.AiDeploymentCreationResponse;
-import com.sap.ai.sdk.core.client.model.AiDeploymentDeletionResponse;
-import com.sap.ai.sdk.core.client.model.AiDeploymentList;
 import com.sap.ai.sdk.core.client.model.AiDeploymentModificationRequest;
 import com.sap.ai.sdk.core.client.model.AiDeploymentModificationRequestWithIdentifier;
-import com.sap.ai.sdk.core.client.model.AiDeploymentModificationResponse;
-import com.sap.ai.sdk.core.client.model.AiDeploymentModificationResponseListInner;
 import com.sap.ai.sdk.core.client.model.AiDeploymentResponseWithDetails;
 import com.sap.ai.sdk.core.client.model.AiDeploymentStatus;
 import com.sap.ai.sdk.core.client.model.AiDeploymentTargetStatus;
 import com.sap.ai.sdk.core.client.model.AiExecutionStatus;
-import com.sap.ai.sdk.core.client.model.RTALogCommonResponse;
```

Comment:
The explicit class names were an implicit test for breaking changes

Reply:
Changes to class names result in *changes of file names*. This would be a better indicator for git-diff hopefully.

According to your logic, it would be best to have a Java file with **all** possible imports(?)

Reply:
The problem with the git diff is that it's invisible between thousand of other changed files when re-generating. So I get the point, there is at least some usefulness to having compile checks.

On the other hand we can't guarantee api stability on the generated code, so it's also not critical. No strong opinion.